test/system/password_reset_test.rb

```diff
@@ -28,15 +28,9 @@ class PasswordResetTest < ActionDispatch::IntegrationTest
       visit new_user_password_path
     end
 
-    after do
-      Devise.mailer.deliveries.clear
-    end
-
-    it 'should error if email does not exist' do
-      assert_no_difference 'Devise.mailer.deliveries.count' do
-        fill_in 'Email', with: 'not_a_real_email@gmail.com'
-        click_button 'Send me password reset instructions'
-      end
+    it 'should respond if email does not exist' do
+      fill_in 'Email', with: 'not_a_real_email@gmail.com'
```

Comment:
Not completely pertinent to the test swap, but why not use an `@example.com` email address? [Example Domain](http://example.com/)

Reply:
You definitely have a point here! Fixed in 22c43b83